src/main/java/org/apache/maven/plugins/gpg/AbstractGpgMojo.java

```diff
@@ -177,7 +188,10 @@ AbstractGpgSigner newSigner(MavenProject project) throws MojoExecutionException,
         signer.setLockMode(lockMode);
         signer.setArgs(gpgArguments);
 
-        loadGpgPassphrase();
+        String passphrase = System.getenv(MAVEN_GPG_PASSPHRASE);
```

Comment:
Shouldn’t the env variable be configurable with the current value as a default ?

Reply:
Fixed